### PR TITLE
octavia monitoring prometheus

### PR DESCRIPTION
**Pull request by fpillotovh**

### CODE REVIEW

File: pages/public_cloud/public_cloud_network_services/technical-resources-02-octavia-monitoring-prometheus/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+```yaml
+[scrape_configs]
+- job_name: 'Octavia LB1'
+  static_configs:
+  - targets: ['192.0.2.10:8088']
+```
+
+For more information on setting up Prometheus, see the [Prometheus project website](https://prometheus.io/).
+
+> [!primary]
```
Comment (yomovh) [inline pages/public_cloud/public_cloud_network_services/technical-resources-02-octavia-monitoring-prometheus/guide.en-gb.md]: To be removed. This part of the documentation is for the administrators (OVHCloud) , not for the end user

File: pages/public_cloud/public_cloud_network_services/technical-resources-02-octavia-monitoring-prometheus/guide.fr-fr.md
Comment (yomovh) [inline pages/public_cloud/public_cloud_network_services/technical-resources-02-octavia-monitoring-prometheus/guide.fr-fr.md:1]: I thought we do not provide anymore french for new public cloud guides ?

File: pages/public_cloud/public_cloud_network_services/technical-resources-02-octavia-monitoring-prometheus/meta.yaml
Comment (yomovh) [inline pages/public_cloud/public_cloud_network_services/technical-resources-02-octavia-monitoring-prometheus/meta.yaml:1]: Octavia is not the name of product, please remove it from the slug and use the official name : public cloud load balancer.

File: pages/index.md
Comment (yomovh) [inline pages/index.md:1]: Please use the official product name "Public Cloud Load Balancer" not octavia

**Review by yomovh — CHANGES_REQUESTED** [review]
See comments

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @fpillotovh

**Pull request merged**